code/_onclick/hud/alert.dm

```diff
@@ -817,3 +817,17 @@ so as to remain in compliance with the most up-to-date laws."
 	master = null
 	screen_loc = ""
 	return ..()
+
+/// Gives the player the option to succumb while in critical condition
+/atom/movable/screen/alert/succumb
+	name = "Succumb"
+	desc = "Shuffle off this mortal coil."
+	icon_state = "succumb"
+
+/atom/movable/screen/alert/succumb/Click()
+	if(!usr || !usr.client)
+		return
+	var/mob/living/living_owner = usr
+	if(!istype(usr))
+		return
+	living_owner.do_succumb(TRUE)
```

Comment:
Does this alert automatically clear after they die? If not, it could be stuck on their hud while dead/a ghost.

Reply:
It seemed to, from my testing. I can double check.

Reply:
Fixed